src/index.html

```diff
@@ -10,8 +10,137 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="bb">
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="sname"> Surname:</label>
+        <input
+
+          type="text"
+          id="sname"
+          name="sname"
+          autocomplete="off"
+          required
+        >
+        <br>
+        <label for="name"> Name:</label>
+        <input
+        class="inp"
+        type="text"
+        id="name"
+        name="name" ><br>
```

Comment:
Переглянь пункти 2 і 3 у чеклісті. Код без відступів важко читати. Також раджу огорнути інпути в лейбли. Пофікси це повсюди. Після фіксів це має виглядати ось так:
```suggestion
        <label for="name"> 
          Name:
          <input
            class="inp"
            type="text"
            id="name"
            name="name"
          >
        </label>
```